suse-module-tools.spec

```diff
@@ -60,8 +60,6 @@ Provides:       udev-extra-rules = 0.3.0
 Obsoletes:      udev-extra-rules < 0.3.0
 Provides:       system-tuning-common-SUSE = 0.3.0
 Obsoletes:      system-tuning-common-SUSE < 0.3.0
-Provides:       sdbootutil-rpm-scriptlets = 2.0
-Obsoletes:      sdbootutil-rpm-scriptlets < 2.0
```

Comment:
Why is this removed?

Reply:
It was wrong. The dependencies must be set for the scriptlets package, not for the main package. I fixed this in "master" already.

Reply:
ah ... and I must have made some mistake rebasing this onto master, after I'd fixed the dependencies in the master branch.